fetch.bs

```diff
@@ -4459,6 +4438,34 @@ the response. [[!HTTP-CACHING]]
 </ol>
 </div>
 
+<div algorithm>
+<p>To <dfn>populate request from client</dfn> given a <a for=/>request</a> <var>request</var>:
+<ol>
+ <li><p>If <var>request</var>'s <a for=request>window</a> is "<code>client</code>", then set
+ <var>request</var>'s <a for=request>window</a> to <var>request</var>'s <a for=request>client</a>,
+ if <var>request</var>'s <a for=request>client</a>'s
+ <a for="environment settings object">global object</a> is a {{Window}} object; otherwise
+ "<code>no-window</code>".
```

Comment:
```suggestion
 <li><p>If <var>request</var>'s <a for=request>window</a> is "<code>client</code>", then: set
 <var>request</var>'s <a for=request>window</a> to <var>request</var>'s <a for=request>client</a>
 if <var>request</var>'s <a for=request>client</a>'s
 <a for="environment settings object">global object</a> is a {{Window}} object; otherwise
 "<code>no-window</code>".
```